calm-visualizer/src/components/graph/Graph.tsx

```diff
@@ -12,84 +12,128 @@ interface Props {
 
 // const groups: string[] = [];
 
-function createConnectsRelationship(instance: BrowserJsPlumbInstance, relationship: CALMConnectsRelationship) {
-    const r = relationship["relationship-type"]["connects"];
-    instance.connect({
-        source: document.getElementById(r.source.node)!,
-        target: document.getElementById(r.destination.node)!,
-        anchor: "Continuous",
-        connector: {
-            type:"Straight",
-            options: {
-                "stub": 25
+function createConnectsRelationship(instance: BrowserJsPlumbInstance, relationships: CALMConnectsRelationship[]) {
+    relationships.map(
+            relationship =>{
```

Comment:
In general I think this method is a bit too long.

I would be in favour of an inner method that does this logic for a single relationship, and then a top level function that maps all the others together. 

Also, file could use an auto-format. @Thels  perhaps we need to do a bit of tightening on our linting rules to avoid these missing spaces and so on?

Reply:
Resolved !